src/ORM/Query.php

```diff
@@ -1299,6 +1299,38 @@ public function insert(array $columns, array $types = [])
         return parent::insert($columns, $types);
     }
 
+    /**
+     * Insert multiple rows of data in one call make use of bulk inserts.
+     *
+     * @param  array  $columns The columns to insert as ['columnName', 'columnName']
+     * @param  array  $values The values to insert as [['columnName' => 'value'], ... ] or ['value', ...]
+     * @param  array  $types A map between columns & their datatypes.
+     * @return $this
+     */
+    public function insertAll(array $columns, array $values, array $types = [])
```

Comment:
I am leaning towards the name `insertMany()`, what do others think?

Reply:
Theoretically, I would agree.
But given that we have updateAll(), deleteAll() as similar operations, and saveMany() etc as callback ops, this seems at least more consistent in the current naming scheme if we use insertAll() here.

Reply:
Well this method is in a different class and Query doesn't have any callbacks.

Reply:
I named it insertAll at first to match other functions like @dereuromark  suggested. That was my first thought too.  insertMany describes it better than insertAll to me. The language difference is subtle. The difference might be explained this way, "I hope every insert() call inserts all the data". This insertMany() function inserts many records. 

I could live with it either way. But the word many describes it better to me.